### DISCUSSION TITLE
Migration from third party media manager

### DISCUSSION THREAD

**Discussion by jasperf** [spatie/laravel-medialibrary#2916] [Q&A]
If we would want to migrate from another media library package are there any steps to reset / re-set the image paths with Laravel Media Library? Anyway to load all existing images from our media disk?

**Comment by PaolaRuby** [comment]
You have to especify the `third party media manager`

**Comment by jasperf** [comment]
@PaolaRuby it would be from [Laravel Media Manager](https://github.com/ctf0/Laravel-Media-Manager).  Working on updating it to work with Laravel 9 and Flysystem 3 but also considering a possible migration if it is too daunting. Seems however that Media Library stores path with all other data in a separate table where we add the path to the JSON data of the module loading the image in a different table and column.
